src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -254,7 +255,7 @@ public void Delete(int id)
 
             if (game == null)
             {
-                throw new ArgumentNullException("game");
+                throw new ArgumentException(nameof(id), "invalid game id");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') ArgumentException constructor arguments have been inverted. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3928)

Reply:
this one

Reply:
yes,